app/views/confirmation.scala.html

```diff
@@ -23,34 +22,39 @@
     appConfig = appConfig,
     bodyClasses = None) {
 
-	  <div class="govuk-box-highlight">
-			@components.heading("confirmation.heading", "heading-xlarge")
-      <p class="font-large">
-    		@messages("confirmation.yourRefNumber")<br>
-        <span id="confirmation-reference" class="bold">@messages("confirmation.refValue", confirmation.reference)</span>
-      </p>
-	  </div>
+    <div class="govuk-box-highlight">
```

Comment:
Does this class do anything?
Generally we don't have access to `govuk-` classes from the design system